apps/web/src/pages/TalentNominations/NominationGroup/components/NominationGroupSidebar.tsx

```diff
@@ -0,0 +1,365 @@
+import { useState } from "react";
+import { useIntl } from "react-intl";
+import PencilSquareIcon from "@heroicons/react/20/solid/PencilSquareIcon";
+
+import {
+  Accordion,
+  Button,
+  CardBasic,
+  Heading,
+  Separator,
+} from "@gc-digital-talent/ui";
+import {
+  FragmentType,
+  getFragment,
+  graphql,
+  TalentNominationGroupStatus,
+} from "@gc-digital-talent/graphql";
+import { commonMessages } from "@gc-digital-talent/i18n";
+import { insertBetween } from "@gc-digital-talent/helpers";
+
+import { getFullNameLabel } from "~/utils/nameUtils";
+import { getClassificationName } from "~/utils/poolUtils";
+
+import NominatedForList from "./NominatedForList";
+
+type AccordionStates = "nominee-contact-information" | "comments" | "";
+
+export const NominationGroupSidebar_Fragment = graphql(/* GraphQL */ `
+  fragment NominationGroupSidebar on TalentNominationGroup {
+    id
+    ...NominationGroupSidebarForList
+    status {
+      value
+      label {
+        localized
+      }
+    }
+    nominee {
+      id
+      email
+      workEmail
+      firstName
+      lastName
+      role
+      telephone
+      preferredLang {
+        label {
+          localized
+        }
+      }
+      currentClassification {
+        id
+        group
+        level
+      }
+      department {
+        name {
+          localized
+        }
+      }
+    }
+    nominations {
+      id
+      nominator {
+        id
+        firstName
+        lastName
+      }
+    }
+  }
+`);
+
+export interface NominationGroupSidebarProps {
+  talentNominationGroupQuery: FragmentType<
+    typeof NominationGroupSidebar_Fragment
+  >;
+}
+
+const NominationGroupSidebar = ({
+  talentNominationGroupQuery,
+}: NominationGroupSidebarProps) => {
+  const intl = useIntl();
+
+  const [accordionState, setAccordionState] = useState<AccordionStates>("");
+
+  const talentNominationGroup = getFragment(
+    NominationGroupSidebar_Fragment,
+    talentNominationGroupQuery,
+  );
+
+  const nominations = talentNominationGroup.nominations ?? [];
+
+  // dialog to go somewhere in span?
+  const nominationsSortedByNominator = nominations.sort((a, b) => {
+    return (
+      (a.nominator?.lastName ?? "").localeCompare(
+        b.nominator?.lastName ?? "",
+      ) ||
+      (a.nominator?.firstName ?? "").localeCompare(b.nominator?.firstName ?? "")
+    );
+  });
+  const nominatorsList = nominationsSortedByNominator.map((nomination) => (
+    <span key={nomination.id}>
+      {getFullNameLabel(
+        nomination.nominator?.firstName,
```

Comment:
This could be a separate component easily. I figured since this will probably get refactored regardless so I just did the quick and dirty way and left it here

Reply:
I think we should do that. When you build components like this they get recreated on every render since react does not know if it and can lead to performance degredation.

https://react.dev/learn/your-first-component#nesting-and-organizing-components

Reply:
Sure, any needed fragments/fragmentation can be done in the later issue
https://github.com/GCTC-NTGC/gc-digital-talent/pull/13100/commits/17cbba7e0ec89d863b3ed244c282fc0f1990f765